### DISCUSSION TITLE
Seeking a RuneLite Plugin for Poison Dynamite Timer

### DISCUSSION THREAD

**Discussion by cheezed** [runelite/runelite#16855] [Ideas]
Hello there. I'm not entirely sure if this is the right place to make a request, but I was wondering if it's possible to create a plugin for RuneLite. The idea is to have a timer that appears for about 30 ticks/18 seconds after using poison dynamite. This would be incredibly helpful for level 3 skillers trying to complete quests or take on foes. If this is the appropriate channel to ask for such a thing, I would be really grateful for any assistance. Thank you!

**Comment by aallen3535** [comment]
Would you mind providing some context as to the purpose of the timer? I'm unfamiliar with this item beyond the wiki description.

**Reply by cheezed** [reply]
I recorded a video to better explain what I was talking about. Let me know if you have any additional questions
https://youtu.be/UKfH1651Q5g
